lib/plain_text.rb

```diff
@@ -0,0 +1,10 @@
+class PlainText < Message
+
+  def encrypt(shifts)
+    encryption = {}
+    encryption[:encryption] = shift_text(shifts)
+    encryption[:key] = ""
+    encryption[:date] = ""
+    encryption
+  end
```

Comment:
PlainText gets the rest of its functionality from the Message class. The plaintext object needs to encrypt the message that it holds but not decrypt.